packages/core/src/modules/credentials/protocol/v2/__tests__/v2credentials-architecture.test.ts

```diff
@@ -108,14 +158,14 @@ describe('V2 Credential Architecture', () => {
       const credFormats: FormatServiceProposeCredentialFormats =
```

Comment:
Can we add an e2e flow where we issue credential with multiple formats? So propose both formats, offer both formats, request both formats, issue both formats